src/libraries/System.Security.Cryptography/tests/HmacSha3_384Tests.cs

```diff
@@ -0,0 +1,235 @@
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Tests
+{
+    public class HmacSha3_384Tests : HmacTests<HmacSha3_384Tests.Traits>
+    {
+        public sealed class Traits : IHmacTrait
+        {
+            public static bool IsSupported => HMACSHA3_384.IsSupported;
+            public static int HashSizeInBytes => HMACSHA3_384.HashSizeInBytes;
+        }
+
+        protected override int BlockSize => 104;
+        protected override int MacSize => HMACSHA3_384.HashSizeInBytes;
+
+        protected override HMAC Create() => new HMACSHA3_384();
+        protected override HMAC Create(byte[] key) => new HMACSHA3_384(key);
+        protected override HashAlgorithm CreateHashAlgorithm() => SHA3_384.Create();
+        protected override byte[] HashDataOneShot(byte[] key, byte[] source) =>
+            HMACSHA3_384.HashData(key, source);
+
+        protected override byte[] HashDataOneShot(ReadOnlySpan<byte> key, ReadOnlySpan<byte> source) =>
+            HMACSHA3_384.HashData(key, source);
+
+        protected override int HashDataOneShot(ReadOnlySpan<byte> key, ReadOnlySpan<byte> source, Span<byte> destination) =>
+            HMACSHA3_384.HashData(key, source, destination);
+
+        protected override bool TryHashDataOneShot(ReadOnlySpan<byte> key, ReadOnlySpan<byte> source, Span<byte> destination, out int written) =>
+            HMACSHA3_384.TryHashData(key, source, destination, out written);
+
+        protected override byte[] HashDataOneShot(ReadOnlySpan<byte> key, Stream source) =>
+            HMACSHA3_384.HashData(key, source);
+
+        protected override byte[] HashDataOneShot(byte[] key, Stream source) =>
+            HMACSHA3_384.HashData(key, source);
+
+        protected override int HashDataOneShot(ReadOnlySpan<byte> key, Stream source, Span<byte> destination) =>
+            HMACSHA3_384.HashData(key, source, destination);
+
+        protected override ValueTask<int> HashDataOneShotAsync(
+            ReadOnlyMemory<byte> key,
+            Stream source,
+            Memory<byte> destination,
+            CancellationToken cancellationToken) => HMACSHA3_384.HashDataAsync(key, source, destination, cancellationToken);
+
+        protected override ValueTask<byte[]> HashDataOneShotAsync(
+            ReadOnlyMemory<byte> key,
+            Stream source,
+            CancellationToken cancellationToken) => HMACSHA3_384.HashDataAsync(key, source, cancellationToken);
+
+        protected override ValueTask<byte[]> HashDataOneShotAsync(
+            byte[] key,
+            Stream source,
+            CancellationToken cancellationToken) => HMACSHA3_384.HashDataAsync(key, source, cancellationToken);
+
+        private static readonly byte[][] s_testKeys = new byte[][]
+        {
+            // From: https://csrc.nist.gov/CSRC/media/Projects/Cryptographic-Standards-and-Guidelines/documents/examples/HMAC_SHA3-256.pdf
```

Comment:
```suggestion
            // From: https://csrc.nist.gov/CSRC/media/Projects/Cryptographic-Standards-and-Guidelines/documents/examples/HMAC_SHA3-384.pdf
```

Reply:
Yep. I confirmed I took the values from the right PDF. (Why is this in a PDF?)